src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        Figure[] list = new Figure [6];

```

Comment:
It's good practice to avoid magic numbers. Consider defining a constant for the array size. For example, `private static final int FIGURE_COUNT = 6;` and then use `new Figure[FIGURE_COUNT];`.